ambry-coordinator/src/test/java/com.github.ambry.coordinator/MockDataNode.java

```diff
@@ -35,36 +32,16 @@
 public class MockDataNode {
   private DataNodeId dataNodeId;
 
-  private Map<BlobId, MaterializedBlob> blobs;
```

Comment:
Now let me explain why we don't need materializedBlob. Earlier, when we had BlobOutput(which had just the InputStream), we need materialized Stream which is nothing but a ByteBuffer of the inputStream. So when getBlob() call is made we return it as ByteBufferInputStream from the materialized stream. With this change, we already have ByteBufferInputStream with us. Why would someone convert from convert from A to B, and then again from B to A. A stands for ByteBufferInputStream and B stands for ByteBuffer. A to B happens during PUT and B to A happens during GET. Hope this clarifies things. If not, we can sync up offline.